### ISSUE TITLE
[tm-bench] compatibility with newer Tendermint (0.11.x and 0.12.x)

### ISSUE THREAD

**Issue by pdobacz** [tendermint/tools#65]
To be able to use `tm-bench` with `v0.11.1` I need to do this slight patch:
https://github.com/ochain/tools/commit/9682c0efae850960af9cdc59f84fca8b8e5631f0
Can submit a PR if its worth it, but maybe better to avoid noise and leap ahead to:

Being compatible with `v0.12.x` requires some more magic, as I'm getting this:
```
user@user-komp:~/sources$ go get github.com/tendermint/tools/tm-bench
# github.com/tendermint/tools/tm-monitor/eventmeter
golang/src/github.com/tendermint/tools/tm-monitor/eventmeter/eventmeter.go:217:28: em.wsc.ResultsCh undefined (type *rpcclient.WSClient has no field or method ResultsCh)
golang/src/github.com/tendermint/tools/tm-monitor/eventmeter/eventmeter.go:230:23: em.wsc.ErrorsCh undefined (type *rpcclient.WSClient has no field or method ErrorsCh)
```

As far as I can tell, both `*Ch`-s got merged into `ResponseCh` [here](https://github.com/tendermint/tendermint/commit/f6adddb4a8e611c2c7d1f8ca64266bda6674e330). I don't know (yet) where to go from here.

**Comment by melekes** [comment]
Have you tried `develop` branch? It should be compatible with tm 0.12

**Comment by melekes** [comment]
What tm-bench version? The latest version should be compatible with TM 0.12

**Comment by pdobacz** [comment]
hmmm... I took `master`, i.e. 30d602d6ed05ba3cf969934937fdcadf7e09e15c. I can't see a `develop` branch in this repo.

**Comment by melekes** [comment]
Have you tried precompiled binaries https://github.com/tendermint/tools/releases or docker builds?

**Comment by pdobacz** [comment]
hmm, nope, I build from source (`go get` and `go install` combinations). I'd love to keep that, as I might want to play around with the code a bit.

**Comment by pdobacz** [comment]
besides the `releases` and `docker build` seem to point to `v1.0.0` - I just tried it, gives the same error.

**Comment by melekes** [comment]
augh, I will take a loop when I have time. Thanks

**Comment by melekes** [comment]
Just released tm-monitor 0.3.1 and tm-bench 0.2.1.
